src/routes/v1/class.route.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const validate = require('../../middlewares/validate');
+const classValidation = require('../../validations/class.validation');
+const classController = require('../../controllers/class.controller');
+const auth = require('../../middlewares/auth');
+
+const router = express.Router();
+
+router.post('/classes', auth(), validate(classValidation.createClass), classController.createClass)
```

Comment:
this should be just '/' since it is added to the `/classes` in the routes/index.js file